source/layers/validation/zes_valddi.cpp

```diff
@@ -2411,15 +2528,17 @@ namespace validation_layer
         zes_fan_handle_t hFan                           ///< [in] Handle for the component.
         )
     {
+        context.logger->log_trace("zesFanSetDefaultMode(hFan)");
```

Comment:
When we log APIs, should we show the hFan value itself? Instead of just the API signature?

Reply:
Lets take this proposal to a future PR.